src/ui/components/ConfirmButton/index.js

```diff
@@ -13,7 +13,7 @@ type Props = {|
   buttonType?: string,
   cancelButtonText?: string | null,
   cancelButtonType?: string,
-  children: React.Element<any>,
+  children: React.Element<any> | string,
```

Comment:
I don't understand why this was needed. Did Flow have a problem with this, or is this just something you noticed? `children` doesn't seem to be passed into any `i18n` functions.

Reply:
Because we use this component that way:

```
<ConfirmButton>{i18n.gettext('submit')}</ConfirmButton>
```

which is perfectly valid, but Flow started to complain that `React.Element<any>` was not compatible with `string`.